plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -863,6 +866,51 @@ public void testTableWithNonNullableColumns()
         assertQuery("SELECT * FROM " + tableName, "VALUES(1, 10, 100), (2, 20, 200)");
     }
 
+    @Test(dataProvider = "changeDataFeedColumnNamesDataProvider")
+    public void testUnsupportedCreateTableWithChangeDataFeed(String columnName)
+    {
+        String tableName = "test_unsupported_create_table_cdf" + randomNameSuffix();
+
+        assertQueryFails(
+                "CREATE TABLE " + tableName + "(" + columnName + " int) WITH (change_data_feed_enabled = true)",
+                "\\QUnable to use %s when change data feed is enabled: [%s]\\E".formatted(CHANGE_DATA_FEED_COLUMN_NAMES, columnName));
+        assertFalse(getQueryRunner().tableExists(getSession(), tableName));
+
+        assertQueryFails(
+                "CREATE TABLE " + tableName + " WITH (change_data_feed_enabled = true) AS SELECT 1 AS " + columnName,
+                "\\QUnable to use %s when change data feed is enabled: [%s]\\E".formatted(CHANGE_DATA_FEED_COLUMN_NAMES, columnName));
+        assertFalse(getQueryRunner().tableExists(getSession(), tableName));
+    }
+
+    @Test(dataProvider = "changeDataFeedColumnNamesDataProvider")
+    public void testUnsupportedAddColumnWithChangeDataFeed(String columnName)
+    {
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "test_add_column", "(col int) WITH (change_data_feed_enabled = true)")) {
+            assertQueryFails(
+                    "ALTER TABLE " + table.getName() + " ADD COLUMN " + columnName + " int",
```

Comment:
add also a test with RENAME COLUMN (the test table should use column mapping mode)
currently not supported, but will ensure we keep desired test coverage when implementing rename column feature

Reply:
#15821 already contains the test to deny RENAME COLUMN with CDF. Let me skip adding the test to this PR.